org/views.py

```diff
@@ -51,4 +53,37 @@ def post(self,request):
             return Response({}, status.HTTP_201_CREATED)
         else:
             data = serializer.errors
-            return Response(data, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(data, status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="list_permissions",
+    method='GET',
+    responses={
+        '200': set_example({}),
+        '400':set_example({"detail": "You are not authorised to view this."}),
+        '401': set_example({"detail": "Authentication credentials were not provided."})
+    }
+)
+@permission_classes([IsAuthenticated])
+@api_view(['GET'])
+def list_permissions_view(request):
+    '''This function shall allow an ADMIN member to see the list of permissions'''
+    
+    members = Member.objects.filter(user=request.user)  # Receiving member status for the respective organisations
+    for member in members:
+        '''
+        User might be a member of various organisations such
+        that it isn't an ADMIN of all, so finding an organisation
+        such that it is an ADMIN member, and printing permission
+        dictionary if it happens to be one.
+        '''                               
+        if member.permissions.permissions.IS_ADMIN:     
```

Comment:
check using the `permissions_to_integer()` function for Admin permissions like
`if member.permissions.permissions.permissions_to_integer() == 31`
I think that will be an easier option.

Reply:
@naveennvrgup sir clarified below that we need not check the admin permission I think, so we can let of this check too I think..?